routes/requests.js

```diff
@@ -60,6 +60,29 @@ router.delete('/:id', isLoggedIn, async (req, res) => {
   })
 })
 
+/**
+ * DELETE /requests/:item
+ *
+ * Removes all requests for a item
+ *
+ * @param id: the request id to delete
+ * @returns the deleted request object
+ */
+ router.delete('/:item/deleteRemaining', isLoggedIn, async (req, res) => {
+  const { item } = req.params
+
+  console.log(req.params) // returns { item: '62e9715ef611b9d04a2cce3f' }
+  // console.log(req.params._id) // undefined
+  // console.log(item._id) // undefined
+  const sameItems = await Request.find({ item: { _id: { $ne: item } } }).populate(['item', 'itemOwner', 'requestor'])
```

Comment:
any idea if we can do this? @phreid

Reply:
I think you can do it with `Request.deleteMany({ item: itemId })` (my syntax might be a bit off). Does this need its own route though? Why not just delete the other requests in the `/accept` route?

Reply:
Added it in, but that request doesn't return the request id's as far as I can see - so I did what I could to get the page to reload with the right cards.